modules/gygax/roll.go

```diff
@@ -71,6 +71,15 @@ func getDiceRollMessage(dice []int) string {
 	return text
 }
 
+func RollDice(message string) string {
+	dice := parseTextForDice(message)
+	if len(dice) < 1 {
```

Comment:
I was thinking this would fall back to the current behavior (implemented in python elsewhere) which just rolls all of them.